contrib/llvm-project-cmake/CMakeLists.txt

```diff
@@ -1,3 +1,39 @@
+set(COMPILER_RT_FUZZER_SRC_DIR "${ClickHouse_SOURCE_DIR}/contrib/llvm-project/compiler-rt/lib/fuzzer")
```

Comment:
1. submodule "llvm-project" contains the entire LLVM repository. We currently only use it to build LLVM libraries for query compilation. Hopefully soon we will also build libcxx/libcxxabi from this submodule. But ideally, all usages reside in different directories in folder "contrib/". So my preference would be if you could move l. 1-36 into "contrib/libfuzzer-cmake/CMakeLists.txt".

2. Do we really want to build libfuzzer unconditionally? I know it is not very large but maybe there could be a CMake option `ENABLE_LIBFUZZER`. Iirc, there will be a special build for fuzzing in CI anyways.